shell/app/modules/project/pages/test-manage/case/header/import-file.tsx

```diff
@@ -35,7 +35,11 @@ const ImportFile = ({ afterImport }: IProps) => {
 
   const onSuccess = ({ successCount }: any) => {
     if (isNumber(successCount)) {
-      message.success(i18n.t('project:imported {total} item successfully', { total: successCount }));
+      if (successCount === 0) {
+        message.success(i18n.t('project:importing'));
```

Comment:
this onSuccess will call twice?

Reply:
It won't. I tried it and it only show me the message once.